Formula/azure-cli@1.0.rb

```diff
@@ -0,0 +1,33 @@
+require "language/node"
+
+class AzureCliAT10 < Formula
```

Comment:
This should match the version. So either we can override `0.10.15` and make it `version "1.0.10.15`, or we can name the formula `azure-cli@0.10` with class name `AzureCliAT010`. Any thoughts on which you prefer?

Reply:
I'll go with `version "1.0.10.15"` then it can be installed with `azure-cli@1.0`

Reply:
OK. And are we sure this shouldn't be azure-cli@1? Is it likely we'll ever have an azure-cli@1.1 formula or azure-cli@2.1 formula?

Reply:
It's possible so I could go with `azure-cli@1` and `azure-cli@2` if that's preferred.

Reply:
I will actually leave this as is so it will be `azure-cli@1.0` and `azure-cli@2.0` with `azure-cli@2.0` pointing to the main `azure-cli`.
There will be no `azure-cli@1.1` in the future so that's not a concern here.

Reply:
I'd prefer we switch to @1 and @2 in that case to be consistent with how we're handling GCC, LLVM, and Node.

Reply:
done.